### ISSUE TITLE
[pkgs] -f option loads .json file instead of .yaml file 

### ISSUE THREAD

**Issue by j-h-scheufen** [monax/monax#1220]
When a .json file with the same name as a .yaml file is present in the directory, even though the `-f` option specifically specified the .yaml file, the .json file is consumed.

To reproduce:
1. Make a simple jobs file and name it MyJobs.yaml
2. Run PM with `-f MyJobs.yaml` and verify that it executes correctly
3. Place an empty file `MyJobs.json` into the same directory
4. Re-run the PM command from 2.) will cause error due to the .json file not being parseable which proves the .json file is used by PM as the jobs file.

```
Sorry, the marmots were unable to load the eris-pm jobs file. Please check your path.
ERROR =>			While parsing config: unexpected end of JSON input
```

Versions:
OSX version: 10.11.6
Eris CLI Version: 0.12.0
Docker:
Client:
 Version:      1.13.0
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Wed Jan 18 16:20:26 2017
 OS/Arch:      darwin/amd64
Server:
 Version:      1.13.0
 API version:  1.25 (minimum version 1.12)
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Wed Jan 18 16:20:26 2017
 OS/Arch:      linux/amd64
 Experimental: false
docker-machine version 0.9.0, build 15fd4c7

**Comment by zramsay** [comment]
this should be addressed **and tested** in #1323

**Comment by zramsay** [comment]
fixed in #1427
